lib/eldap/src/eldap.erl

```diff
@@ -346,6 +398,12 @@ Add an entry. The entry must not exist.
 ```
 """.
 -doc(#{since => <<"OTP R15B01">>}).
+-spec add(Handle, Dn, Attributes) -> ok | {ok, Value} | {error, Reason} when
+    Handle :: handle(),
+    Dn :: string(),
+    Attributes :: [attribute()],
+    Value :: term(),
```

Comment:
According to the old doc this function can also return "{ok, {referral, referrals()}}",
which '`Value`" must reflect.

Reply:
fixed